modules/33acrossBidAdapter.js

```diff
@@ -267,10 +307,66 @@ function _buildBannerORTB(bidRequest) {
     ? _getViewability(element, utils.getWindowTop(), minSize)
     : NON_MEASURABLE;
 
+  const ext = contributeViewability(viewabilityAmount);
+
   return {
     format,
-    ext: contributeViewability(viewabilityAmount)
+    ext
+  }
+}
+
+// BUILD REQUESTS: VIDEO
+// eslint-disable-next-line no-unused-vars
+function _buildVideoORTB(bidRequest) {
+  const videoAdUnit = utils.deepAccess(bidRequest, 'mediaTypes.video', {});
+  const videoBidderParams = utils.deepAccess(bidRequest, 'params.video', {});
+
+  const video = {}
+
+  const sizes = videoAdUnit.playerSize || [];
+  const {w, h} = _getSize(sizes);
+  video.w = w;
+  video.h = h;
+
+  // Obtain all ORTB params related video from Ad Unit
+  VIDEO_ORTB_PARAMS.forEach((param) => {
+    if (videoAdUnit.hasOwnProperty(param)) {
+      video[param] = videoAdUnit[param];
+    }
+
+    // Override with bidder specific video params
+    if (videoBidderParams.hasOwnProperty(param)) {
```

Comment:
maybe merge the videoAdUnit and videoBidderParams together, then do this assignment

Reply:
I have done so as per your feedback.